token/light_picker.js

```diff
@@ -2,6 +2,8 @@ function tokenUpdate(data) {
   canvas.tokens.controlled.map(token => token.update(data));
 }
 
+let torchAnimation = {"type": "torch", "speed": 1, "intensity": 1};
```

Comment:
IMHO, speed and intensity are better on '2' but it is really opinionated and the user can modify to his taste in the script.